### DISCUSSION TITLE
Green Champion

### DISCUSSION THREAD

**Discussion by WBurggraaf** [Green-Software-Foundation/discussions#147] [Ask Anything]
How can you follow your https://champions.greensoftware.foundation/ submission? It is now a kinda silent process and after submission a bit untransparent. Thanks for your time!

**Comment by russelltrow** [comment]
Hey @WBurggraaf, we had to temporarily pause the Champions administration work due to Carbon Hack and we've been waiting for a new GSF PM to start who will get everything moving again. You should see updates again soon, thank you for your patience.
